src/base/baseBrushTool.js

```diff
@@ -0,0 +1,331 @@
+import external from './../externalModules.js';
+import EVENTS from './../events.js';
+import baseTool from './../base/baseTool.js';
+// Utils
+import isToolActive from '../fancy-tools/shared/isToolActive.js';
+import { getNewContext } from '../util/drawing.js';
+import { COLOR_MAP_ID } from '../stateManagement/brushToolColors.js';
+
+export default class extends baseTool {
```

Comment:
For searchability, can we include the name baseBrushTool when exporting?

Reply:
I'll go through and do this for all the tools as we approach the end, as well as change classes to be capitalized.